internal/serdeutil/raw_message.go

```diff
@@ -0,0 +1,31 @@
+package serdeutil
```

Comment:
Could you explain why it is named like this? It is the first time I see that phrase :)

Reply:
serde is short for serialization/deserialization, I know of Java and Rust which use this nomenclature